gammapy/datasets/simulate.py

```diff
@@ -647,7 +647,7 @@ def __init__(
         n_event_bunch=10000,
         dataset_kwargs=None,
     ):
-        self.dataset_kwargs = dataset_kwargs
+        self.dataset_kwargs = dataset_kwargs if dataset_kwargs else {}
```

Comment:
Can we simply utilise the following:
```suggestion
        self.dataset_kwargs = dataset_kwargs or {}
```